src/child.rs

```diff
@@ -195,5 +205,7 @@ mod ffi {
     extern {
         pub fn pivot_root(new_root: *const c_char, put_old: *const c_char)
             -> c_int;
+
+        pub fn setns(fd: c_int, nstype: c_int) -> c_int;
```

Comment:
Ah, sorry, we don't use `nix` in `child.rs` anyway (there is a comment about that). But `libc::setns` should be fine.